src/pydata_sphinx_theme/locale/fr/LC_MESSAGES/sphinx.po

```diff
@@ -1,22 +1,24 @@
-# French translations for PROJECT.
-# Copyright (C) 2023 ORGANIZATION
-# This file is distributed under the same license as the PROJECT project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
-#
+# English translations for pydata-sphinx-theme.
```

Comment:
```suggestion
# French translations for pydata-sphinx-theme.
```
??

Reply:
I know, these headers are killing me. They are copied from the .pot file (english) and then slightly modified by transifex. apparently this line didn't get that chance. 

from a functional point of view, it changes nothing